grid2op/Agent/RandomRedispatchAgent.py

```diff
@@ -0,0 +1,77 @@
+import numpy as np
+from grid2op.Agent import BaseAgent
+
+class RandomRedispatchAgent(BaseAgent):        
+    def __init__(self, action_space,
+                 n_gens_to_redispatch=2,
+                 redispatching_increment=1.0):
+        """
+        Agent constructor
+
+        Parameters
+        ----------
+        :action_space: :class:`grid2op.Action.ActionSpace`
+             the Grid2Op action space
+
+        :n_gens_to_redispatch: `int`
+          The maximum number of dispatchable generators to play with 
+
+        :redispatching_increment: `float`
+          The redispatching MW value used in both directions
+        """
+        super().__init__(action_space)
+        self.desired_actions = []
+
+        # Get all generators IDs
+        gens_ids = np.arange(self.action_space.n_gen, dtype=int)
+        # Filter out non resipatchable IDs
+        gens_redisp = gens_ids[self.action_space.gen_redispatchable == True]
```

Comment:
remove the "== True"
`gens_redisp = gens_ids[self.action_space.gen_redispatchable]`
evaluates faster and is more readable